ports/nrf/boards/sparkfun_nrf52840_mini/pins.c

```diff
@@ -0,0 +1,117 @@
+#include "shared-bindings/board/__init__.h"
+
+#include "board_busses.h"
+
+STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
+    { MP_ROM_QSTR(MP_QSTR_P0_00), MP_ROM_PTR(&pin_P0_00) },
+    { MP_ROM_QSTR(MP_QSTR_P0_01), MP_ROM_PTR(&pin_P0_01) },
```

Comment:
No need to add the internal nRF names here. They are available through `microcontroller.pin` automatically. This table should just include names to match a board's silkscreen and other common designations (like SDA and SCL). Limiting whats in here makes `dir(BOARD)` easier to understand.

Reply:
Sweet. More than happy to delete some stuff in here!